src/amp-story-player/amp-story-player-impl.js

```diff
@@ -630,6 +649,28 @@ export class AmpStoryPlayer {
     this.isLaidOut_ = true;
   }
 
+  /**
+   * Fetches more stories from the publisher's endpoint.
+   * @return {!Promise}
+   * @private
+   */
+  fetchStories_() {
+    const url = this.readPlayerConfig_().behavior.endpoint;
```

Comment:
Nit: `this.playerConfig_`

Reply:
Reason why I used this.readPlayerConfig_() in all of these is because it returns the cached this.playerConfig_ if it has already been initialized or it initializes it as a preventive measure.

But since we are reading it at the buildCallback() now it should be fine either way.